### DISCUSSION TITLE
Hide an element on app

### DISCUSSION THREAD

**Discussion by pyinstallerbaggins** [marimo-team/marimo#2276] [Q&A] [answered]
How can one supress this section of the app
![image](https://github.com/user-attachments/assets/c5404490-c6ba-4341-a318-877f5eca794a)

When I click it navigates to markdowns
![image](https://github.com/user-attachments/assets/83164dd2-3c75-48e3-a36e-6c5997e91dcc)

**Comment by mscolnick** [comment]
That is the table of contents. It is not currently configurable to be made hidden.

**Reply by mscolnick** [reply, marked as answer]
We do allow custom css, so you can maybe hide it through that.
